test-app/tests/integration/helpers/file-queue-helper-test.gts

```diff
@@ -20,6 +20,31 @@ module('Integration | Helper | file-queue', function (hooks) {
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
+  test('service can be used before helper', async function (assert) {
```

Comment:
Until some code fixing the addon is pushed, this test **will** fail. And that is ok as this PR is **only** intended to show the bug. I don't think I will have time anytime soon to fix the code itself, so very happy if someone can take-over and finish this.